### PR TITLE
[fix bug 1303857] Add ravioli firstrun promo.

### PR DESCRIPTION
**Pull request by jpetto**
## Description

Adds a Yahoo Search promotion first run page for funnelcake testing.

Two updates yet to come, but should not block review:

1. ✅ ~~Better quality high-res Katie Couric image~~
2. ✅ ~~Actual funnelcake id (89 was invented for testing)~~

## Bugzilla link

https://bugzilla.mozilla.org/show_bug.cgi?id=1303857

## Testing

Test `/en-US/firefox/49.0/firstrun/?f=89` in Fx >= 40.0.

## Checklist
- [ ] Related functional & integration tests passing.

### CODE REVIEW

File: bedrock/firefox/templates/firefox/firstrun/ravioli.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  </h2>
+
+  <h1>{{ _('Search and see, with Yahoo Search.') }}</h1>
+
+  <p>{{ _('Yahoo Search brings you relevant results with high quality images and video for a more visual search experience.') }}</p>
+
+  <figure class="ravioli">
+    {{ high_res_img('firefox/firstrun/ravioli.png', {'alt': 'Yahoo Search'}) }}
+    <figcaption>
+      {{ _('Yahoo is one of Mozilla’s search partners. <a rel="external" href="%s">Learn more.</a>')|format('https://blog.mozilla.org/blog/2015/11/11/a-new-yahoo-search-experience-for-firefox-users-in-u-s/') }}
```
Comment (craigcook) [inline bedrock/firefox/templates/firefox/firstrun/ravioli.html]: Split this into two strings? Then "Learn more." can be its own string without the HTML. I know this is English only for now but might this eventually be localized?

Comment (jpetto) [inline bedrock/firefox/templates/firefox/firstrun/ravioli.html]: Yeah, couldn't hurt to split into two strings.

**Comment by jpetto** [comment]
Still **Do Not Merge**, but only waiting on green light from Winston.

**Comment by jpetto** [comment]
I believe this has the green light for tomorrow (9/28). If needed, I put the branch up in this repo:

https://github.com/mozilla/bedrock/tree/bug-1303857-firstrun-ravioli-katie-couric-test

**Comment by alexgibson** [comment]
Closing in favor of #4374 so we can squash the commits.

**Pull request closed without merging**